jupyterhub/spawners.py

```diff
@@ -255,6 +256,12 @@ def get_pod_manifest(self):
                 options.get('commit_sha')
         }
 
+        # add username to labels
+        safe_username = escapism.escape(self.user.name, escape_char='-').lower()
```

Comment:
I didn't want to create a dependency to `renku_notebooks` for this file because it is used in JupyterHub. Do you think it is OK to have that dependency?

Reply:
Actually, I've tried it and does not seem to work. The Hub pod cannot start because it cannot import this spawner.

Reply:
yep, you are right that won't work. We will tackle this when we refactor this code into its own package.